Source/WebKit/GPUProcess/GPUConnectionToWebProcess.h

```diff
@@ -375,6 +382,9 @@ class GPUConnectionToWebProcess
 #if ENABLE(VIDEO)
     RefPtr<RemoteMediaResourceManager> m_remoteMediaResourceManager WTF_GUARDED_BY_CAPABILITY(mainThread);
     Ref<RemoteMediaPlayerManagerProxy> m_remoteMediaPlayerManagerProxy;
+#endif
+#if ENABLE(LINEAR_MEDIA_PLAYER)
+    VideoReceiverEndpointManager m_videoReceiverEndpointManager;
```

Comment:
Worth lazy allocating?

Reply:
I had a style error (it doesn't handle the #ifdef in the constructor thinking I'm indenting a namespace)